include/linux/soundwire/sdw.h

```diff
@@ -637,6 +637,7 @@ struct sdw_link_ops;
 /**
  * struct sdw_master_device - SoundWire 'Master Device' representation
  * @dev: Linux device for this Master
+ * @bus: Bus handle
```

Comment:
I will add an empty line between line 638 and 639 which is also a request from Vinod.

Reply:
it's not required. The kernel-doc format does not specify an extra line, in fact when @andy-shev reviewed my changes on kernel-doc errors he asked me to remove the extra lines.